spoon-bytecode/pom.xml

```diff
@@ -0,0 +1,476 @@
+<!--
+
+    Copyright (C) 2006-2017 INRIA and contributors
+    Spoon - http://spoon.gforge.inria.fr/
+
+    This software is governed by the CeCILL-C License under French law and
+    abiding by the rules of distribution of free software. You can use, modify
+    and/or redistribute the software under the terms of the CeCILL-C license as
+    circulated by CEA, CNRS and INRIA at http://www.cecill.info.
+
+    This program is distributed in the hope that it will be useful, but WITHOUT
+    ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+    FITNESS FOR A PARTICULAR PURPOSE. See the CeCILL-C License for more details.
+
+    The fact that you are presently reading this means that you have had
+    knowledge of the CeCILL-C license and that you accept its terms.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>fr.inria.gforge.spoon</groupId>
+        <artifactId>spoon-pom</artifactId>
+        <version>7.2.0-SNAPSHOT</version>
+        <relativePath>../spoon-pom</relativePath>
+    </parent>
+
+    <groupId>fr.inria.gforge.spoon</groupId>
+    <artifactId>spoon-bytecode</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0-SNAPSHOT</version>
+    <name>Spoon Bytecode</name>
+    <description>Extension of spoon for bytecode through decompilation</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>fr.inria.gforge.spoon</groupId>
+            <artifactId>spoon-core</artifactId>
+            <version>7.2.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
The version of junit used should be the same in all subprojects of Spoon. So you should define the version in spoon-pom properties.